inspirehep/config.py

```diff
@@ -125,6 +125,114 @@ def _(x):
 USERPROFILES_EXTEND_SECURITY_FORMS = False
 USERPROFILES_SETTINGS_TEMPLATE = 'inspirehep_theme/accounts/settings/profile.html'
 
+# Collections
+# ===========
+COLLECTIONS_DELETED_RECORDS = '{dbquery} AND NOT collections.primary:"DELETED"'
+"""Enhance collection query to exclude deleted records."""
+
+COLLECTIONS_QUERY_PARSER = 'inspirehep.modules.search.parser:Main'
+"""User search query lexical parser."""
+
+COLLECTIONS_QUERY_WALKERS = [
+    'inspirehep.modules.search.walkers.pypeg_to_ast:PypegConverter',
+]
+"""Modules to create the query AST."""
+COLLECTIONS_USE_PERCOLATOR = False
+"""Define which percolator you want to use.
+
+Default value is `False` to use the internal percolator.
```

Comment:
This docstring is copy/pasted from `invenio-collections`. Maybe the phrase is a bit confusing since they called it 'internal percolator'. If we think of `percolator` just as the ES one, then the config makes sense `COLLECTIONS_USE_PERCOLATOR` `True` or `False`